addons/medical_damage/ACE_Medical_Injuries.hpp

```diff
@@ -48,11 +51,13 @@ class ACE_Medical_Injuries {
                 minDamage = 0.3;
                 maxDamage = 0.6;
                 bleedingRate = 0.02;
+                lethalities[] = {{1, 0.10}, {0, 0.20}};
```

Comment:
Not part of this PR, but we will probably want to add macros for the body part indices to get rid of the magic numbers.